package.json

```diff
@@ -43,13 +45,14 @@
     "test": "ng test",
     "test:libs": "concurrently \"ng test core --code-coverage\" \"ng test storefrontlib --code-coverage\" \"ng test organization --code-coverage\" \"ng test storefinder --code-coverage\" \"ng test smartedit --code-coverage\" \"ng test qualtrics --code-coverage\" \"ng test product --code-coverage\" \"ng test product-configurator --code-coverage\" \"ng test cdc --code-coverage\" \"ng test setup --code-coverage\"",
     "test:storefront:lib": "ng test storefrontlib --sourceMap --code-coverage",
+    "build:ssr": "env-cmd --no-override -e dev,b2c,$SPA_ENV ng run storefrontapp:server:production",
```

Comment:
Can you help me understand, why we need a flag `--no-override`  in `yarn build:ssr` and `yarn start`, but we don't need it in `yarn build` or `yarn start:prod`?

Reply:
It was introduced by problems with Windows from @marlass and @Pucek9 . It seems like those were sufficient.